doc/news/changes.h

```diff
@@ -526,6 +526,12 @@ inconvenience this causes.
 
 
 <ol>
+  <li> Fixed: inverse_operator() now populates <code>Tvmult</code> and
+  <code>Tvmult_add</code> correctly.
+  <br>
+  (Jean-Paul Pelteret, David Wells, Matthias Maier, 2015/12/30)
+  </li>
```

Comment:
Is this a change against 8.3, or was all of this introduced only post 8.3? In the latter case no entry is needed here.